src/applications/claims-status/utils/appeals-v2-helpers.jsx

```diff
@@ -258,9 +258,9 @@ export function addStatusToIssues(issues) {
  * @returns {object} One appeal object or undefined if not found in the array
  */
 export function isolateAppeal(state, id) {
-  return _.find(
+  return find(
```

Comment:
is `find` missing an import?

Reply:
Yup, looks like I missed this when I was grepping.